tcms/core/management/commands/refresh_permissions.py

```diff
@@ -0,0 +1,91 @@
+from django.apps import apps
+from django.contrib.auth.models import Group, Permission
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = ('Refresh permissions for Tester group  and remove '
+            'stale ones. Use -v 0 to suppress most of the '
+            'console output'
+            )
+
+    @staticmethod
+    def get_all_permissions():
+        """
+            Returns a set of all available permissions in format
+            app_label.perm_codename
+        """
+        permissions = list()
+        for perm in Permission.objects.all():
+            permissions.append("%s.%s" %
+                               (perm.content_type.app_label, perm.codename))
+        return set(permissions)
+
+    @staticmethod
+    def get_all_installed_apps():
+        """
+            Returns a set of currently installed applications as app_labels
+        """
+        applications = list()
+        for app_config in apps.get_app_configs():
+            applications.append(app_config.label)
+        return set(applications)
+
+    @staticmethod
+    def assign_default_group_permissions(assign_if_none=False):
+        """
+            Assigns default permissions for Kiwi TCMS apps to Tester group
+        """
+        admin = Group.objects.get(name='Administrator')
+        if assign_if_none and admin.permissions.count() == 0:
+            all_perms = Permission.objects.all()
+            admin.permissions.add(*all_perms)
+
+        tester = Group.objects.get(name='Tester')
+        # Assign permissions for Kiwi TCMS related applications
```

Comment:
The entire block for tester group permissions is now missing the conditional checks. They should remain as before + add the new one into the mix.